example/android/app/build.gradle

```diff
@@ -147,8 +147,8 @@ android {
         applicationId "com.twilio.verify.reactnative.example"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 1
-        versionName "1.0"
+        versionCode 3
+        versionName "0.3.0"
```

Comment:
same here

Reply:
It had a wrong version, the sample app should have the same version as the SDK to be tested easily